koku/api/report/aws/serializers.py

```diff
@@ -215,8 +215,12 @@ class AWSEC2ComputeFilterSerializer(BaseFilterSerializer):
     limit = None
     offset = None
 
-    resource_id = StringOrListField(child=serializers.CharField(), required=False)
-    instance_name = StringOrListField(child=serializers.CharField(), required=False)
+    # Not valid for this endpoint.
+    # instance_name and resource_id is filtered by instance field.
+    instance_name = None
+    resource_id = None
```

Comment:
We don't need to do this (these can be removed). `instance_name` and `resource_id` are not defined in the `BaseFilterSerializer` class. We only need to set fields to `None` when we are overriding the base class.